package.json

```diff
@@ -240,7 +240,7 @@
       "^.+\\.(ts|tsx|js|jsx)$": "babel-jest",
       "\\.graphql$": "jest-raw-loader"
     },
-    "testRegex": "/__tests__/.+?\\.test\\.(ts|tsx|js|jsx)$",
+    "testRegex": "/__tests__/.+?\\.test\\.(ts|tsx)$",
```

Comment:
I think we could simplify this further by removing the folder designation -- the file extension should do it, `"testRegex": "(.test)\\.(ts|tsx)$"`

Reply:
True, but I think keeping the folder designation will help ensure proper folder structure, so I'd lean on keeping it personally.

Reply:
Fair enough, but it also means that if someone names the folder wrong, it doesn't get run -- we should find some way to check folder naming on pre-commit or something in the future... Force had a related issue recently where many directories were never being run because of typos like no plural "s" in "tests"

Reply:
That could be a Peril rule.

Reply:
@alloy I'd be down to pair on this with you!

Reply:
File a ticket and let’s do it (next week?)

Reply:
https://artsyproduct.atlassian.net/browse/PLATFORM-1543